docs/LIBRARY.md

```diff
@@ -88,6 +88,9 @@ artifacts in a map of filenames to `io.Reader`. The `preflight.log` is written
 to the ArtifactWriter for library callers, and can be accessed once checks have
 been executed.
 
+A logger of choice should be added to the context with logr.NewContext. Otherwise,
+no logs will be available from the library.
+
```

Comment:
The code snippet likely also needs to be updated, because it currently reflects a preflight.log being written to the artifact writer and I don't think that's the case anymore with this patch.